presto-record-decoder/src/test/java/io/prestosql/decoder/avro/TestAvroDecoder.java

```diff
@@ -146,12 +166,9 @@ private static String getAvroSchema(Map<String, String> fields)
     {
         GenericData.Record record = new GenericData.Record(schema);
         values.forEach(record::put);
-        try {
-            DataFileWriter<GenericRecord> dataFileWriter = new DataFileWriter<>(new GenericDatumWriter<>(schema));
-
+        try (DataFileWriter<GenericRecord> dataFileWriter = new DataFileWriter<>(new GenericDatumWriter<>(schema))) {
```

Comment:
This should be fixed in the other commit.

Reply:
This is in a separate commit: https://github.com/prestosql/presto/pull/1602/commits/9763db5048ccddeec1a73efc706709ee4ef24700

Or do you want to have it in a separate PR?

Reply:
Done